fusor-ember-cli/app/components/base-f.js

```diff
@@ -28,5 +28,11 @@ export default Ember.Component.extend({
 
   unitsClassSize: Ember.computed(function () {
     return this.getWithDefault('unitsSize', 'col-md-2');
-  })
+  }),
+
+  actions: {
+    doNothing() {
+      return false;
```

Comment:
@cfchase, @eriknelson, it seemed to work even without action `doNothing`, but I think it's more explicit to have it more. I guess it does the same as `event.preventDefault` in jQuery.